index.js

```diff
@@ -15,3 +15,67 @@ closeUp.addEventListener('click', closing);
 closePort.addEventListener('click', closing);
 closeAbout.addEventListener('click', closing);
 closeCont.addEventListener('click', closing);
+
+// Form Validation
+
+const nameError = document.querySelector('#name-err');
+const emailError = document.querySelector('#email-err');
+const textError = document.querySelector('#text-err');
+const submitError = document.querySelector('#submit-err');
+
+function validateName() {
+  const name = document.querySelector('#name-id').value;
+
+  if (name.length === 0) {
+    nameError.innerHTML = 'Name is required';
+    return false;
+  }
+
+  if (!name.match(/^[A-Za-z]*\s{1}[A-Za-z]*$/)) {
+    nameError.innerHTML = 'Wrie full name';
+    return false;
+  }
+  nameError.innerHTML = '<i class="fa-regular fa-circle-check" style="color: #3bc456;"></i>';
+  return true;
+}
+
+function validateEmail() {
+  const email = document.querySelector('#email-id').value;
+
+  if (email.length === 0) {
+    emailError.innerHTML = 'Email is required';
+    return false;
+  }
+
+  if (!email.match(/^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|.(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/)) {
+    emailError.innerHTML = 'Invalid email';
+    return false;
+  }
```

Comment:
- [ ] Nice work implementing the validation, it looks really good :heart_eyes: However it requested in the project requirement that the form is submit when the email field contains only lowercase(see image). Kindly include that logic in you validation to follow the project requirement.
![Screenshot from 2023-06-23 14-38-25](https://github.com/2004-okari/MY-PORTFOLIO/assets/91142432/085f6eca-690d-4a9f-b4c3-4ad232f0e04b)
### Your project validation :point_down: 
![Screenshot from 2023-06-23 14-29-08](https://github.com/2004-okari/MY-PORTFOLIO/assets/91142432/a6935ad4-0469-48b7-accc-eceb8523c760)